VirtualReality/Widgets/qMRMLVirtualRealityHomeWidget.cxx

```diff
@@ -106,6 +113,55 @@ vtkMRMLVirtualRealityViewNode* qMRMLVirtualRealityHomeWidget::virtualRealityView
   return d->VirtualRealityViewNode;
 }
 
+//-----------------------------------------------------------------------------
+void qMRMLVirtualRealityHomeWidget::setupModuleButton(QWidget* widget, QIcon* icon)
+{
+  Q_D(const qMRMLVirtualRealityHomeWidget);
+
+  QPushButton* moduleButton = new QPushButton(d->ModulesGroupBox);  
+  d->ModulesGroupBoxLayout->addWidget(moduleButton);
+  moduleButton->setIcon(*icon);
+  
+  widget->setParent(d->ModuleWidgetFrame);
+  this->ModuleWidgetsMap[moduleButton] = widget;
+  
+  QObject::connect(moduleButton, SIGNAL(clicked()), this, SLOT(onModuleButtonPressed()));
+  QObject::connect(d->backButton, SIGNAL(clicked()), this, SLOT(onBackButtonPressed()));
+ 
+  moduleButton->setVisible(true);
+  widget->setVisible(false);
+  d->backButton->setVisible(false);
+}
+
+//-----------------------------------------------------------------------------
+void qMRMLVirtualRealityHomeWidget::onModuleButtonPressed()
+{
+  Q_D(const qMRMLVirtualRealityHomeWidget);
```

Comment:
No need for const here because the function is not a const.

You would only need the const if it was declared like this:
void onModuleButtonPressed()const;